core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/filebeat.yml

```diff
@@ -99,6 +99,8 @@
       vars:
         existing_output_es_password: >-
           {{ _filebeat_existing_config['output.elasticsearch'].password }}
+        existing_setup_dashboards: >-
+          {{ _filebeat_existing_config['setup.dashboards'] }}
```

Comment:
```suggestion
          {{ _filebeat_existing_config['setup.dashboards'] | default({}) }}
```

Reply:
This node is not defined in epicli v0.8 and v0.9.